edx_rest_framework_extensions/tests/test_permissions.py

```diff
@@ -155,3 +182,176 @@ def test_has_permission(self, enforce_scopes, authentication_class, jwt_filters,
             permissions.JwtHasContentOrgFilterForRequestedCourse().has_permission(request, view),
             expected_result,
         )
+
+
+@ddt.ddt
+class JwtHasUserFilterForRequestedUserTests(TestCase):
+    """ Tests for JwtHasUserFilterForRequestedUserTests permission class. """
+
+    def _create_request(self, user_filters, requested_username):
+        url = '/?username={}'.format(requested_username)
+        request = RequestFactory().get(url)
+        request.user = UserFactory(username='this_user')
+        request.auth = generate_jwt(request.user, filters=user_filters)
+        return request
+
+    @ddt.data(
+        # no user filters
+        ([], 'this_user', True),
+        ([], 'another_user', True),
+
+        # accessing self
+        (['user:me'], 'this_user', True),
+        (['user:this_user'], 'this_user', True),
+
+        # accessing another
+        (['user:me'], 'another_user', False),
+        (['user:this_user'], 'another_user', False),
+        (['user:another_user'], 'another_user', True),
+    )
+    @ddt.unpack
+    def test_has_permission(self, user_filters, requested_username, expected_result):
+        request = self._create_request(user_filters, requested_username)
+        self.assertEqual(
+            permissions.JwtHasUserFilterForRequestedUser().has_permission(request, None),
+            expected_result,
+        )
+
+
+@ddt.ddt
+class JwtRestrictionApplicationOrUserAccessTests(TestCase):
```

Comment:
ApplicationRestriction?

Reply:
typo: change to JwtRestrictedApplicationOrUserAccessTests

Reply:
done.